src/rviz/selection/forwards.h

```diff
@@ -76,6 +76,10 @@ inline uint32_t colorToHandle(Ogre::PixelFormat fmt, uint32_t col)
   {
     handle = col >> 8;
   }
+  else if (fmt == Ogre::PF_R8G8B8)
```

Comment:
Maybe add another or-clause to the first if?

Reply:
Here is also for consistency I think.

Reply:
well, the first if statement does exactly the same as this one and already has two conditions. But it doesn't matter much I suppose.